source/common/certificate_provider/certificate_provider_manager_impl.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <string>
+#include "absl/container/node_hash_map.h"
+#include "envoy/config/core/v3/extension.pb.h"
+#include "envoy/certificate_provider/certificate_provider_manager.h"
+
+namespace Envoy {
+namespace CertificateProvider {
+
+/**
+ * A manager for certificate provider instances.
+ */
+class CertificateProviderManagerImpl: public CertificateProviderManager {
+public:
+
+  CertificateProviderManagerImpl();
+
+  void addCertificateProvider(std::string /*name*/,
+                              const envoy::config::core::v3::TypedExtensionConfig& /* config */) override;
+
+  CertificateProviderSharedPtr getCertificateProvider(std::string /* name */) override;
```

Comment:
No need to comment out parameter name in declarations (throughout)
```suggestion
  CertificateProviderSharedPtr getCertificateProvider(std::string name) override;
```

Reply:
got it